pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers=
+    dsl
+    coredeps
```

Comment:
where is this used?

Reply:
nowhere in this PR, but several other places. pytest emits a warning if you haven't defined the markers upfront.